tdms/tests/unit/field_tests/test_TDFieldExporter2D.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file test_TDFieldExporter2D.cpp
+ * @brief Unit tests for the TDFieldExporter2D class.
+ */
+#include "field.h"
+
+#include <catch2/catch_test_macros.hpp>
+
+TEST_CASE("TDFieldExporter2D") {
+
+  // create a 0-field for us to export
+  const int I_tot = 8, J_tot = 1, K_tot = 8;
+  ElectricSplitField Es(I_tot, J_tot, K_tot);
+  Es.allocate_and_zero();
+
+  // create our field exporter
+  TDFieldExporter2D tdfe2d;
+  tdfe2d.folder_name = ".";
```

Comment:
```suggestion
  tdfe2d.folder_name = "."; // TODO: Sam has some temp directory code that should be used here. TDMS/#80.
```
This means we need write permission to the current working directory to run the tests?

Reply:
Afraid so - couldn't think of a suitable place to put the output that is going to be produced. If you have some `temp` directory stuff, I agree we can hijack that later.

Reply:
When this is merged I'll cherry-pick it out of my feature branch and see if I can make a small self-contained PR.